modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/admin/throttlingpolicy/ThrottlePolicyExportImportTestCase.java

```diff
@@ -0,0 +1,597 @@
+                keyTemplate);
+
+        ApiResponse<CustomRuleDTO> addedPolicy;
+        try {
+            //Add the custom throttling policy
+            addedPolicy = restAPIAdmin.addCustomThrottlingPolicy(CustomPolicyDTO);
+
+            //Assert the status code and policy ID
+            Assert.assertEquals(addedPolicy.getStatusCode(), HttpStatus.SC_CREATED);
+            CustomRuleDTO addedPolicyDTO = addedPolicy.getData();
+            customPolicyId = addedPolicyDTO.getPolicyId();
+            Assert.assertNotNull(customPolicyId, "The policy ID cannot be null or empty");
+            CustomPolicyDTO.setPolicyId(customPolicyId);
+            CustomPolicyDTO.setIsDeployed(true);
+            //Verify the created custom throttling policy DTO
+            adminApiTestHelper.verifyCustomThrottlePolicyDTO(CustomPolicyDTO, addedPolicyDTO);
+        } catch (ApiException e) {
+            if (userMode == TestUserMode.TENANT_ADMIN) {
+                Assert.assertEquals(e.getCode(), HttpStatus.SC_FORBIDDEN);
+            }
+        }
+    }
+
+    @Test(groups = {"wso2.am"}, description = "Exported Sample Custom ThrottlePolicy" )
+    public void testCustomThrottlePolicyExport() throws Exception {
+        if (userMode == TestUserMode.TENANT_ADMIN) {
+            throw new SkipException("Skipping the export custom policy");
+        }
+        exportedFileCustomPolicy = exportArtifact(user.getUserName(), user.getPassword(), customPolicyName,
+                customPolicyType);
+        CustomPolicyDTO.setIsDeployed(false);
+        CustomPolicyDTO.setType("CustomRule");
+        ExportThrottlePolicyDTO customPolicyExportedDTO = DtoFactory.createExportThrottlePolicyDTO(THROTTLE_POLICY_TYPE,
+                CUSTOM_POLICY_SUBTYPE, APIM_VERSION, CustomPolicyDTO);
+
+        StringBuilder contentBuilder = new StringBuilder();
+        try (Stream<String> stream = java.nio.file.Files.lines(Paths.get(String.valueOf(exportedFileCustomPolicy)),
+                StandardCharsets.UTF_8)) {
+            stream.forEach(s -> contentBuilder.append(s).append("\n"));
+        } catch (IOException e) {
+            throw new APIManagerIntegrationTestException(
+                    "Error in reading from extracted Throttle Policy file " + exportedFileCustomPolicy, e);
+        }
+
+        String exportedThrottlePolicyContent = contentBuilder.toString();
+        JSONParser parser = new JSONParser();
+        JSONObject exportedThrottlePolicyJson = (JSONObject) parser.parse(exportedThrottlePolicyContent);
+        CustomPolicyDTO.setIsDeployed(false);
+        CustomPolicyDTO.setType("CustomRule");
+        customPolicyExportedDTO = DtoFactory.createExportThrottlePolicyDTO(THROTTLE_POLICY_TYPE, CUSTOM_POLICY_SUBTYPE,
+                APIM_VERSION, CustomPolicyDTO);
+
+        Gson gson = new Gson();
+        ObjectMapper mapper = new ObjectMapper();
+        ExportThrottlePolicyDTO expectedExportedPolicy = gson.fromJson(exportedThrottlePolicyJson.toJSONString(),
```

Comment:
Can declare and initialize in same line

Reply:
Fixed with 3106d801961d2d8e82a5a35117427b8337a92455